From d0327e3325837b4aa22198ac090530a3060952ef Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 9 Jan 2008 07:16:42 +0000
Subject: [PATCH] rblsmtpd.c: don't use a the default rbl.maps.vix.com if run without -r switch.

rbl.maps.vix.com is obsolete, so don't use it as default rbl if run
without the -r switch; default to no rbl instead.
---
 rblsmtpd.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/rblsmtpd.c b/rblsmtpd.c
index cc8ba2e..9695d9e 100644
--- a/rblsmtpd.c
+++ b/rblsmtpd.c
@@ -155,7 +155,6 @@ void rblsmtpd(void)
 
 main(int argc,char **argv,char **envp)
 {
-  int flagwantdefaultrbl = 1;
   char *x;
   int opt;
 
@@ -182,7 +181,7 @@ main(int argc,char **argv,char **envp)
       case 'c': flagfailclosed = 1; break;
       case 'C': flagfailclosed = 0; break;
       case 't': scan_ulong(optarg,&timeout); break;
-      case 'r': rbl(optarg); flagwantdefaultrbl = 0; break;
+      case 'r': rbl(optarg); break;
       case 'a': antirbl(optarg); break;
       default: usage();
     }
@@ -190,7 +189,6 @@ main(int argc,char **argv,char **envp)
   argv += optind;
   if (!*argv) usage();
 
-  if (flagwantdefaultrbl) rbl("rbl.maps.vix.com");
   if (decision >= 2) rblsmtpd();
 
   pathexec_run(*argv,argv,envp);
-- 
1.6.0.3

